functions/prompty/moderationOpenAi/openAiModeration.js

```diff
@@ -1,13 +1,13 @@
-const { Configuration, OpenAIApi } = require("openai");
-
-const configuration = new Configuration({
-  apiKey: process.env.OPENAI_API_KEY,
-});
-
-module.exports.openAiModeration = async function (prompt) {
-  const openai = new OpenAIApi(configuration);
-  const moderation = await openai.createModeration({
-    input: prompt,
-  });
-  return moderation.data.results[0];
-};
+const { Configuration, OpenAIApi } = require("openai");
+
+const configuration = new Configuration({
+  apiKey: process.env.OPENAI_API_KEY,
+});
+
+module.exports.openAiModeration = async function (prompt) {
+  const openai = new OpenAIApi(configuration);
+  const moderation = await openai.createModeration({
+    input: prompt,
+  });
+  return moderation.data.results[0];
+};
```

Comment:
ideally, moderation and text completion would be combined into one function in the backend. As you have it now, it requires two calls to the backend — rather, one function which returns `{ error: true/false }`  and the payload would probably make it more efficient and safer.

Admittedly, if someone is technically savvy enough to isolate the`TextCompletion` call from the `Moderation` call, they're probably not a high school student. But, you never know. And we don't want to accidentally create a proxy to ChatGPT (given that it's banned in some districts)